### ISSUE TITLE
Provide a "Please publish to open-vsx" issue template

### ISSUE THREAD

**Issue by SamB** [EclipseFdn/open-vsx.org#1246]
It would be quite helpful to provide a template for people to use when requesting that people publish their extensions to open-vsx.org, with a quick explanation of why it's important, and link(s) to more details on why and how to do it.

When I think about submitting issues like that, I always find myself spending a lot of time trying to figure out what pages to send the extension author(s) to, and in the end I typically don't arrive at a satisfactory answer and don't submit an issue at all.

**Comment by kineticsquid** [comment]
@SamB Want to make sure I understand your suggestion. An author publishes their extension(s). Are you suggesting a template for the information about an extension they publish, e.g. the information on this page: https://open-vsx.org/extension/redhat/java?

**Comment by filiptronicek** [comment]
Hey @SamB 👋! Me and the team made one in the [publish-extensions repo](https://github.com/open-vsx/publish-extensions) - it can be found at https://github.com/open-vsx/publish-extensions/blob/master/docs/external_contribution_request.md. Is this what you are looking for?

**Comment by filiptronicek** [comment]
We should probably add a link to this doc from the https://open-vsx.org Wiki [[1](https://github.com/EclipseFdn/open-vsx.org/wiki)] cc @kineticsquid

**Comment by kineticsquid** [comment]
@SamB Can you confirm this is what you're looking for?

@filiptronicek I agree we should include this on the wiki. I'm thinking we'll want this up front because it's an early fork in the publishing road. 

E.g. either
```
Home

Welcome to the Eclipse Open VSX Wiki! Read the following pages to get started.

    [Publishing your extensions](https://github.com/eclipse/openvsx/wiki/Publishing-Extensions)
    [Publishing others' extensions](https://github.com/eclipse/openvsx/wiki/Publishing-Others-Extensions)
    [Namespaces in Open VSX](https://github.com/eclipse/openvsx/wiki/Namespace-Access)
    [Using Open VSX in VS Code](https://github.com/eclipse/openvsx/wiki/Using-Open-VSX-in-VS-Code)
    [Deployment documentation](https://github.com/eclipse/openvsx/wiki/Deploying-Open-VSX)
    [Registry API documentation](https://github.com/eclipse/openvsx/wiki/Registry-API)
```

Or (at `[Publishing your extensions](https://github.com/eclipse/openvsx/wiki/Publishing-Extensions)` link):
```
How to Publish an Extension

This guide describes publishing to the public registry at [open-vsx.org](https://open-vsx.org/), which is now managed by the [Eclipse Foundation](https://www.eclipse.org/) ([see the announcement](https://blogs.eclipse.org/post/brian-king/open-vsx-registry-under-new-management)).

Similar steps may apply to registries that are hosted elsewhere, but make sure to pass the correct URL to the [ovsx](https://www.npmjs.com/package/ovsx) tool.

If this is your extension ..., if you want to publish an extension authored by another ...
```

**Comment by filiptronicek** [comment]
@kineticsquid I would vote for the second option, since if I wanted to publish an extension as an author, I would visit the publishing page, and, if I were to be just a community member with an extension in mind, I would still visit the `How to Publish an Extension` page, because in both cases, that's what I want to do!

**Comment by kineticsquid** [comment]
@filiptronicek How's this:

**How to Publish an Extension**

This guide describes publishing to the public registry at [open-vsx.org](https://open-vsx.org/), which is now managed by the [Eclipse Foundation](https://www.eclipse.org/) ([see the announcement](https://blogs.eclipse.org/post/brian-king/open-vsx-registry-under-new-management)).

Similar steps may apply to registries that are hosted elsewhere, but make sure to pass the correct URL to the [ovsx](https://www.npmjs.com/package/ovsx) tool.

If you are the author of the extension you wish to publish, proceed with the steps below. If you are not the author, we suggest you first reach out to the author with an issue in their GitHub repo to request that they publish their extension to [open-vsx.org](https://open-vsx.org/). We've drafted a [template](https://github.com/open-vsx/publish-extensions/blob/master/docs/external_contribution_request.md) with suggested content for the issue.

**Comment by filiptronicek** [comment]
@kineticsquid sounds great!

**Comment by kineticsquid** [comment]
Hmm... I don't appear to have the authority to update the wiki. Can I get access?

**Comment by fredg02** [comment]
If you are referring to the GitHub wiki, you should be able to update it, since you have write permissions.

**Comment by kineticsquid** [comment]
@fredg02 I can edit these pages, https://github.com/EclipseFdn/open-vsx.org/wiki, but not this page, https://github.com/eclipse/openvsx/wiki/Publishing-Extensions. Maybe I'd need to be a committer?

**Comment by fredg02** [comment]
Correct, you would need to be a committer to change https://github.com/eclipse/openvsx/wiki/Publishing-Extensions

**Comment by filiptronicek** [comment]
Looks like fortunately I've got access to that Wiki, so made the change myself if that's ok (can revert if needed!): https://github.com/eclipse/openvsx/wiki/Publishing-Extensions

**Comment by kineticsquid** [comment]
Looks good! @SamB?
